Blockzilla/AddSearchEngineViewController.swift

```diff
@@ -18,8 +19,9 @@ class AddSearchEngineViewController: UIViewController, UITextViewDelegate {
     private var templateInput = UITextView()
     private var templatePlaceholderLabel = UILabel()
     
-    init(delegate: AddSearchEngineDelegate) {
+    init(delegate: AddSearchEngineDelegate, searchEngineManager: SearchEngineManager) {
```

Comment:
@boek What do you think about injecting `SearchEngineManager` vs adding isValidTemplate a method to the delegate

Reply:
Dependency Injection 👍 (but still would like to hear from @boek ).